lib/everypolitician_extensions.rb

```diff
@@ -38,8 +38,23 @@ def memberships_at_end
       end
     end
   end
+
+  module MembershipExtension
+    # The group that a membership represents
```

Comment:
This is a slightly odd description. A membership doesn't really _represent_ a Group (or Area in version below). 

I'd suggest something like:
* The political group (party / faction) this membership is held on behalf of.